modules/client/examples/01-client/02-deposit-eth.ts

```diff
@@ -1,34 +1,32 @@
 /* MARKDOWN
-### Depositing ETH to a DAO
-
-Handles the flow of depositing the native token to an Aragon DAO.
+### Deposit ETH to a DAO
 
+Handles the flow of depositing the native EVM token (when in mainnet, it's ETH) to an Aragon OSx DAO.
```

Comment:
The token deposited does not depend on the network you are, it depends on the token type that you select on the params, if its native i'll be ETH, if its erc20 it will be any ERC20 token with its corresponding address

Reply:
nevermind just saw that they are 2 diferent files for each type of deposit, resolving